roles/StackStorm.st2/handlers/main.yml

```diff
@@ -8,27 +8,32 @@
 - name: reload st2
   become: yes
   command: st2ctl reload --register-all
+  when: "'st2actionrunner' in st2_services"
```

Comment:
Listing st2actionrunner might be very specific to implementation details in your environment.
Technically both st2api and st2sensorcontainer should have access to pack content as well.

Reply:
The `st2api` service is "leaky" though. Given the way that this ansible setup works, you also need to install `st2api` on any host that has `st2chatops` because it is used to create the ChatOps API key.

Reply:
could maybe do something like:
```
when: "'st2actionrunner' in st2_services and 'st2api' in st2_services"
```

Reply:
How about `st2api or st2actionrunner or st2sensorcontainer`? Technically, all these services should have access to the pack content.